packages/snaps-controllers/src/services/iframe/IframeExecutionService.test.browser.ts

```diff
@@ -167,4 +167,58 @@ describe('IframeExecutionService', () => {
       'Failed to access document of the snap iframe: SecurityError',
     );
   });
+
+  it('confirms that events are secured', async () => {
```

Comment:
Can this be moved to `BaseSnapExecutor.test.browser.ts`, since that's where we have similar tests to these as well?

Reply:
The lockdown is applied in `iframe/index.ts`, so I don't think that would work.